ethcore/src/client/client.rs

```diff
@@ -293,11 +321,12 @@ impl<V> Client<V> where V: Verifier {
 
 		{
 			if !good_blocks.is_empty() && self.block_queue.queue_info().is_empty() {
```

Comment:
@gavofyork Is the `block_queue` is empty check needed here?
My understanding is that if we don't send the event - some of the blocks' transactions will not be removed from queue because of that (cause in next event the blocks won't be there)